Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,66 @@ class Program
     {
         static void Main(string[] args)
```

Comment:
It still needs to be done for 2 people. I recommend splitting the contents of main into a function of its' own and simply calling it twice.

Reply:
I created a new function called 'AskUserQuestionsAndUserPrintProfile' is this naming good?
Also I I put the function underneath all the other functions. Is this a good place to have it?